service-report/src/main/kotlin/org/veupathdb/service/mblast/report/service/mblast/MBlastQuerySvc.kt

```diff
@@ -0,0 +1,74 @@
+package org.veupathdb.service.mblast.report.service.mblast
+
+import com.fasterxml.jackson.databind.node.ObjectNode
+import org.apache.logging.log4j.LogManager
+import org.gusdb.fgputil.Tuples.TwoTuple
+import org.veupathdb.lib.compute.platform.job.JobStatus
+import org.veupathdb.lib.hash_id.HashID
+import org.veupathdb.lib.jackson.Json
+import org.veupathdb.service.mblast.report.ServiceOptions
+import java.io.InputStream
+import java.lang.Exception
+import java.net.URI
+import java.net.http.HttpClient
+import java.net.http.HttpRequest
+import java.net.http.HttpResponse
+
+object MBlastQuerySvc {
```

Comment:
Having `Client` in the name here might be helpful, though I guess it should be obvious since it's in the report service.

Reply:
Updated